### PR TITLE
Schedule zziplib only on SLE 12 SP3 and later

### PR DESCRIPTION
**Pull request by martinsmarcelo**
poo#50153 

Tests in:
SLES 12 Sp2 - http://10.161.229.197/tests/369#step/zziplib/2
SLES 12 Sp4 - http://10.161.229.197/tests/371#step/zziplib/1

### CODE REVIEW

File: tests/console/zziplib.pm [hunk 1]
```diff
@@ -17,37 +17,45 @@ use strict;
...
-    # Option -v  creates a core dump(bsc#1129403), create a soft-failure
-    my $RETURN_CODE = script_run("unzip-mem -v $filezip");
-    if (($RETURN_CODE) eq "0") {
-        assert_script_run("unzip-mem -v $filezip");
-    } else {
-        record_soft_failure "bsc#1129403 - Option -v creates a core dump";
-        save_screenshot;
+
+    # SLES 12 SP1 and Sp2 need add repo SDK manually, if case, skip test.
+    if (is_sle('>12-SP2')) {
```
Comment (czerw) [inline tests/console/zziplib.pm]: Please do it in main_common.pm in similar way  https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/41c61212eb62f9d09aa2284fc8ebf9c0fedabaca/lib/main_common.pm#L1611

**Comment by czerw** [comment]
Please update the commit message to something more reasonable (you can check https://chris.beams.io/posts/git-commit/).

File: tests/console/zziplib.pm (hunk 1 above)
Comment (martinsmarcelo) [reply tests/console/zziplib.pm]: Dammit!!! ok, changed. Thank you.

**Comment by martinsmarcelo** [comment]
> Please update the commit message to something more reasonable (you can check https://chris.beams.io/posts/git-commit/).

ok, I will pay attention about this

File: lib/main_common.pm [hunk 2]
```diff
@@ -1600,7 +1600,7 @@ sub load_extra_tests_console {
     loadtest 'console/procps';
     loadtest "console/lshw" if ((is_sle('15+') && (check_var('ARCH', 'ppc64le') || check_var('ARCH', 'x86_64'))) || is_opensuse);
     loadtest 'console/quota'   unless is_jeos;
-    loadtest 'console/zziplib' unless is_jeos;
+    loadtest 'console/zziplib' if is_sle('12-SP3+');
```
Comment (czerw) [inline lib/main_common.pm]: Looks like `is_jeos` condition was removed by accident. Please extend condition to don't run it on JeOS.

Comment (martinsmarcelo) [reply lib/main_common.pm]: done!

**Comment by czerw** [comment]
@martinsmarcelo  thanks for changes, but commit messages still needs some attention,

**Comment by martinsmarcelo** [comment]
> @martinsmarcelo thanks for changes, but commit messages still needs some attention,

**Comment by dzedro** [comment]
Into commit message don't add our #poo tags, they have no value here just shortly describe PR like "Schedule zziplib only on SLE 12 SP3 and later due to missing dependencies" and poo#50153 can be in commit body or something like that

**Comment by martinsmarcelo** [comment]
> Into commit message don't add our #poo tags, they have no value here just shortly describe PR like "Schedule zziplib only on SLE 12 SP3 and later due to missing dependencies" and poo#50153 can be in commit body or something like that

Humm, ok, understood now .

**Pull request merged**